.gitignore

```
@@ -2,6 +2,8 @@
 .cache/
 .DS_Store
 .env
+.idea
+.vscode/*
```

Nitpicking: use the same folder path format throughout the file
```suggestion
.idea/
.vscode/
```